src/Common/src/System/Collections/Generic/LargeArrayBuilder.cs

```diff
@@ -50,12 +50,21 @@ public LargeArrayBuilder(bool initialize)
         /// Use <see cref="Add"/> if adding to the builder is a bottleneck for your use case.
         /// Otherwise, use <see cref="SlowAdd"/>.
         /// </remarks>
+        public void Add(T item) => Add(item, limit: int.MaxValue);
```

Comment:
I think I found in my experiments a while ago that the JIT inlines methods sequentially; it will recognize the body of `Add(T)` is small, then inline it. Then it will recognize the body of `Add(T, int)` is small, and inline it. Consequently, you only need to apply AggressiveInlinine once.